language_service/src/compilation.rs

```diff
@@ -69,6 +72,13 @@ impl Compilation {
             language_features,
         );
 
+        let lints = qsc::linter::run_lints(&unit, Some(lints_config));
```

Comment:
We should do this for notebooks as well (though, feel free to open a new issue/PR for this so as not to bloat this one any further). It should be super easy - just insert a corresponding call into the `new_notebook` function after we build up the `CompileUnit` for the notebook.

Reply:
Filed an issue as a remainder.